HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -40,11 +40,19 @@ def customiseFor17792(process):
             producer.produceSeedStopReasons = cms.bool(False)
     return process
 
+def customiseFor18118(process):
+    hbhereconames = ['hltHbhePhase1Reco','hltHbherecoMethod2L1EGSeeded','hltHbherecoMethod2L1EGUnseeded']
+    for hbhereconame in hbhereconames:
+        if hasattr(process,hbhereconame):
+            getattr(process,hbhereconame).saveEffectivePedestal = cms.bool(False)
+    return process
```

Comment:
Please check (if hasattr) whether the parameter ```saveEffectivePedestal``` is already set to some value (and then presumably do not overwrite it!)